.github/actions/release/bump-versions/action.yml

```diff
@@ -45,63 +45,35 @@ runs:
       run: |
         cargo set-version ${{ inputs.version }}
 
-    # cargo workspaces ignores examples_legacy/ but cargo release still tries to version it during publishing.
-    - name: Bump Rust examples_legacy version
-      shell: bash
-      if: ${{inputs.release-target == 'rust'}}
-      working-directory: examples_legacy
-      run: |
-        cargo set-version ${{ inputs.version }}
-
     # cargo workspaces ignores the legacy crates because they are excluded from the workspace.
     # Bump versions for consistency.
```

Comment:
Do we still need to do this?

Reply:
Thanks for pointing that out, that can go indeed.